assets/src/js/admin/admin-settings.js

```diff
@@ -112,14 +112,16 @@ jQuery( document ).ready( function( $ ) {
 
 		// Bailout if admin can not edit notification status setting.
 		if ( ! parseInt( $this.data( 'edit' ) ) ) {
-			// Remove all notice.
 			$( 'div.give-email-notification-status-notice' ).remove();
 
 			// Add notice.
 			$( 'hr.wp-header-end' ).after( '<div class="updated error give-email-notification-status-notice"><p>' + $( this ).closest( '.give-email-notification-status' ).data( 'notice' ) + '</p></div>' );
 
 			// Scroll to notice.
-			$( 'html, body' ).animate( { scrollTop: $( 'div.give-email-notification-status-notice' ).position().top }, 'slow' );
+			var noticeContainer = $( 'div.give-email-notification-status-notice' );
```

Comment:
I this still even used? I'm not seeing a notice to scroll to...

This isn't a blocker, since this prevents the error as expected - just wondering if we can get rid of it all together.

Reply:
@kjohnson Good catch.

I found that a notice displays if an email notification is not editable when the admin clicks on the lock icon. I added a fix. It will appear now.

https://github.com/impress-org/givewp/compare/05b24836230c...31e59f031fec https://github.com/impress-org/givewp/compare/31e59f031fec...4009d2c0f776